vlasovsolver/cpu_acc_map.cpp

```diff
@@ -320,10 +320,10 @@ bool map_1d(SpatialCell* spatial_cell,
          firstBlockIndexK = (firstBlockIndexK < max_v_length ) ? firstBlockIndexK : max_v_length - 1;
          lastBlockIndexK  = (lastBlockIndexK  >= 0)            ? lastBlockIndexK  : 0;
          lastBlockIndexK  = (lastBlockIndexK  < max_v_length ) ? lastBlockIndexK  : max_v_length - 1;
-         if(firstBlockIndexK < Parameters::bailout_velocity_space_wall_margin
-            || firstBlockIndexK >= max_v_length - Parameters::bailout_velocity_space_wall_margin
-            || lastBlockIndexK < Parameters::bailout_velocity_space_wall_margin
-            || lastBlockIndexK >= max_v_length - Parameters::bailout_velocity_space_wall_margin
+         if(firstBlockIndexK < static_cast<int>(Parameters::bailout_velocity_space_wall_margin)
+            || firstBlockIndexK >= max_v_length - static_cast<int>(Parameters::bailout_velocity_space_wall_margin)
+            || lastBlockIndexK < static_cast<int>(Parameters::bailout_velocity_space_wall_margin)
+            || lastBlockIndexK >= max_v_length - static_cast<int>(Parameters::bailout_velocity_space_wall_margin)
```

Comment:
Again, ugh. Massive amounts of static_casts.
Are you sure this isn't more easily solved by leaving max_v_length as an unsigned number?

Reply:
("A cast doesn't make the code more correct. It only makes the warning go away.")

Reply:
Right you are. In this case the casts are especially messy, and a proper solution would be to change to signed types in `definitions.h` which is probably out of scope of this PR.